src/common

```diff
@@ -150,6 +150,64 @@ else
   $(error Invalid setting for BUILD_ENV: $(BUILD_ENV_))
 endif
 
+# select default solver to be minisat2 if no other is specified
+ifndef IPASIR
+ifndef PRECOSAT
+ifndef PICOSAT
+ifndef LINGELING
+ifndef CHAFF
+ifndef BOOLEFORCE
+ifndef MINISAT
+ifndef MINISAT2
+ifndef GLUCOSE
+MINISAT2 = ../../minisat-2.2.1
```

Comment:
I am not sure this can be achieved in a different way, as you only want to set the default path in case none of the other variables is set already.